### PR TITLE
[stable/filebeat] raised filebeat version to 6.4.0

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**:
* raises filebeat version to 6.4.0

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/filebeat/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: A Helm chart to collect Kubernetes logs with filebeat
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: filebeat
 version: 1.0.0
```
Comment (davidkarlsen) [inline stable/filebeat/Chart.yaml]: please bump the chart version too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/filebeat/values.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: docker.elastic.co/beats/filebeat-oss
-  tag: 6.3.2
+  tag: 6.4.0
```
Comment (davidkarlsen) [inline stable/filebeat/values.yaml:3]: please also update readme to match

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/filebeat/values.yaml (hunk 2 above)
Comment (monotek) [reply stable/filebeat/values.yaml:3]: readme was already changed ;-)

File: stable/filebeat/Chart.yaml (hunk 1 above)
Comment (monotek) [reply stable/filebeat/Chart.yaml]: done :-)

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**